OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -168,6 +176,115 @@ int main()
 //==============================================================================
 // Test Cases
 //==============================================================================
+void testSmith2016Ligament()
+{
+    using namespace SimTK;
+
+    double mass = 1;
+    double stiffness = 10;
+    double restlength = 0.5;
+    double dissipation = 0.1;
+    double start_h = 0.5;
+
+    // Setup OpenSim model
+    Model osimModel{};
+    osimModel.setName("Smith2016Ligament");
+    //OpenSim bodies
+    const Ground& ground = osimModel.getGround();;
+    OpenSim::Body pulleyBody("PulleyBody", mass ,Vec3(0), 
+                             mass*SimTK::Inertia::brick(0.1, 0.1, 0.1));
+    OpenSim::Body block("block", mass ,Vec3(0), 
+                        mass*SimTK::Inertia::brick(0.2, 0.1, 0.1));
+    block.attachGeometry(new Brick(Vec3(0.2, 0.1, 0.1)));
+    block.scale(Vec3(0.2, 0.1, 0.1), false);
+    
+    WrapCylinder* pulley = new WrapCylinder();
+    pulley->set_radius(0.1);
+    pulley->set_length(0.05);
+
+    // Add the wrap object to the body, which takes ownership of it
+    pulleyBody.addWrapObject(pulley);
```

Comment:
We should use the wrap or remove it. I realize PathSpring does NOT use the wrap, but that should also be addressed (separately).